astropy/coordinates/builtin_frames/altaz.py

```diff
@@ -103,12 +103,12 @@ class AltAz(BaseCoordinateFrame):
     default_representation = r.SphericalRepresentation
     default_differential = r.SphericalCosLatDifferential
 
-    obstime = TimeAttribute(default=None)
-    location = EarthLocationAttribute(default=None)
-    pressure = QuantityAttribute(default=0, unit=u.hPa)
-    temperature = QuantityAttribute(default=0, unit=u.deg_C)
-    relative_humidity = QuantityAttribute(default=0, unit=u.dimensionless_unscaled)
-    obswl = QuantityAttribute(default=1 * u.micron, unit=u.micron)
+    obstime = TimeAttribute(default=None, doc="The reference time (e.g., time of observation)")
```

Comment:
I think (but have not checked) that the `sphinx` aspect of this could also be solved by just adding a docstring right below, i.e., 
```
obstime = TimeAttribute(default=None)
"""The reference time (e.g., time of observation)."""
```
Note that I think your solution is a lot nicer, so this is really just for completeness!